README.md

```diff
@@ -59,14 +50,22 @@ For the CI tests we only create specific files for the environment (see `tests/t
 
 ### Running the role
 
-Run the playbook with a version of ansible higher than 1.8:
+Replace the target information (user, IPADDRESS) and run the playbook with a version of ansible higher than 1.8:
 
-    $ ansible-playbook -e "pipelining=True" -b -u ubuntu --private-key=~/.ssh/id_rsa -i ./inventory.txt playbook.yml
+    $ ansible-playbook -b -u ubuntu -i 'IPADDRESS,' playbook.yml
```

Comment:
@awailly What about replacing `ubuntu` with `USER` as you did for the IP address?
It might not be obvious for everyone that it's the user here ;)